_docs/concepts/security/authn-policy.md

```diff
@@ -0,0 +1,101 @@
+
+- **source.principal**: peer principal. If peer authentiation is not used, the attribute is not set.
+- **request.auth.principal**: depends on the policy principal binding, this could be peer principal (if USE_PEER) or origin principal (if USE_ORIGIN).
+- **request.auth.audiences**: reflect the audience (`aud`) claim within the origin-JWT (JWT that is used for origin authentication)
+- **request.auth.presenter**: similarly, reflect the authorized presenter (`azp`) claim of the origin-JWT.
+- **request.auth.claims**: all raw string claims from origin-JWT.
+
+Origin principal (principal from origin authentication) is not explicitely output. In general, it can always be reconstructed by joining (`iss`) and subject (`sub`) claims with a "/" seperator (for example, if `iss` and `sub` claims are "*googleapis.com*" and "*123456*" respectively, then origin principal is "*googleapis.com/123456*"). On the other hand, if principal binding is USE_ORIGIN, **request.auth.principal** carries the same value as origin principal.
+
+
+## Anatomy of the policy
+
+### Target selectors
+
+Defines rule to find service(s) on which policy should be applied. If no rule is provided, the policy is matched to all services in the namespace, so-called namespace-level policy (as opposed to service-level policies which have non-empty selector rules). Istio uses the service-level policy if available, otherwise it falls back to namespace-level policy. If neither is defined, it uses the default policy based on service mesh config and/or service annotation, which can only set mutual TLS setting (these are pre-0.7 mechanisms to config mutual TLS for Istio service mesh). See [testing Istio nutual TLS]({{home}}/docs/tasks/security/mutual-tls.html) and [per-service mutual TLS enablement]({{home}}/docs/tasks/security/per-service-mtls.html) for more details.
+
+Operators are responsible for avoiding conflicts, e.g create more than one service-level policy that matches to the same service(s) (or more than one namespace-level policy on the same namespace).
+
+
+Example: rule to select product-page service (on any port), and reviews:9000.
+
+```
+ targets:
+ - name: product-page
+ - name: reviews
+   ports:
+   - number: 9000
+```
+
+### Peer authentication
+
+
+Defines authentication methods (and associated parameters) that are supported for peer authentication. It can list more than one method; only one of them needs to be satisfied for the authentication to pass. However, starting with the 0.7 release, only mutual TLS is supported. Omitting this if peer authentication is not needed.
+
+
+Example of peer authentiation using mutual TLS:
+
+```
+  peers:
+  - mtls:
+```  
+
+> Starting with the  0.7 release, mtls settings doesn't require any parameter (hence `- mtls:` or `- mtls: null` declaration is sufficient). In future, it may carry arguments to provide different mTLS implementations.
```

Comment:
the 0.7 release -> Istio 0.7
mtls settings setting -> the `mtls` setting
parameter -> parameters

Reply:
Done